src/cf-layout.less

```diff
@@ -1605,6 +1605,121 @@
     }
 }
 
+/* topdoc
+  name: Featured content module
+  family: cf-layout
+  patterns:
+    - name: .featured-content-module
+      markup:
+        <section class="block
+                        block__border
+                        block__flush
+                        featured-content-module">
```

Comment:
Should we move away from combining blocks with other content in preparation to shifting to atomic design? In atomic design the `featured-content-module` organism wouldn't know about it's margin, whereas the template would. This would mean this part would look like:

```
<section class="block
                block__border
                block__flush">
    <div class="featured-content-module">
```